adminer/static/editing.js

```diff
@@ -283,6 +283,30 @@ function editingRemoveRow(button, name) {
 	return true;
 }
 
+/** Move table row for field
+* @param HTMLInputElement
+* @param bool direction
+* @return boolean
+*/
+function editingMoveRow(button, dir){
+	var row = parentTag(button, 'tr');
+	var table = row.parentNode;
+	var tags = table.getElementsByTagName('tr');
+	for (var i=0; i < tags.length; i++) {
```

Comment:
I'd have to use `nextElementSibling`/`previousElementSibling` which would drop support for IE8 and below - is that ok?